crates/router/src/configs/settings.rs

```diff
@@ -115,6 +115,13 @@ pub struct Settings<S: SecretState> {
     pub events: EventsConfig,
     #[cfg(feature = "olap")]
     pub connector_onboarding: SecretStateContainer<ConnectorOnboarding, S>,
+    pub white_list_incoming_header: WhiteListIncomingHeaders,
```

Comment:
Can we call this "allow list"?

Reply:
Sounds good, even `unmasked_headers` should be fine.

Reply:
cool, will make the required change